Mod Menu Helper/assets/modmenu/lang/tr_tr.json

```diff
@@ -3,42 +3,49 @@
   "modmenu.dropInfo.line2": "diğerleri desteklenmemektedir.",
   "modmenu.summaryTranslation.advancementinfo": "Daha büyük ve akıllı başarım ekranı\n§9✎§8 Renkler, bilgi kutucuk büyüklüğü, ...",
   "modmenu.summaryTranslation.advancements-enlarger": "§c*§r Daha büyük başarım ekranı\n§8✎ (seçenekleri yok)",
-  "modmenu.summaryTranslation.animatica": "OptiFine'ın hareketli dokular\n§8✎ (sadece gelişmiş seçenekler)",
-  "modmenu.summaryTranslation.antighost": "Hayalet blokları /ghost\n§9✎§8 tuş atamasıyla düzelt.",
-  "isxander-main-menu-credits.antighost": "Hayalet blokları /ghost",
+  "modmenu.summaryTranslation.animatica": "OptiFine'ın hareketli dokuları\n§8✎ (sadece gelişmiş seçenekler)",
+  "modmenu.summaryTranslation.antighost": "Hayalet blokları /ghost ile düzelt\n§9✎§8 Tuş ataması",
+  "isxander-main-menu-credits.antighost": "Hayalet blokları /ghost ile düzelt",
   "modmenu.summaryTranslation.betterbeds": "§c*§r Hızlı yataklar\n§8✎ (seçenekleri yok)",
   "modmenu.summaryTranslation.bettermounthud": "Atın üstündeyken yemek ve XP'yi göster\n§8✎ (seçenekleri yok)",
   "modmenu.summaryTranslation.bobby": "§c*§r Sunuculardaki yüklü yığınları yüklü tutar\n§9✎§8 Varlıkları blokla, yığınları azalt, önbelleği koru, ...",
   "modmenu.summaryTranslation.borderlessmining": "Tam ekrandayken Alt+Tab desteği (opt-in)\n§9✎§8 Modu, macOS desteğini, özel boyut vb. aç/kapa",
```

Comment:
Is "opt-in" intended to be the same in Turkish? I probably missed it on the first translation but since you're revising it anyway, might as well let you know.

Reply:
I'm positive that we don't use "opt-in" as it is. As far as I checked, direct meanings of "opt-in" are "getting involved", "registering", "signing up", "choose to be involved", "to prefer" etc. Any of these matches with or close to the meaning of the original context? If it doesn't, what does it indicate exactly?

Reply:
The meaning here is intended to be "disabled by default - optional". You may use just "optional" as well.